### DISCUSSION TITLE
have 1 hour that im waiting my github pages be deployed

### DISCUSSION THREAD

**Discussion by carloMorais** [community/community#48921] [Other Feature Feedback, Questions, & Ideas] [answered]
i was refreshing the url that github give me and have at lest 1 hour
[https://delfo2.github.io/Calculadora-em-Typescript/](url github pages)
still something else that i have to do before ?
![image](https://user-images.githubusercontent.com/105376328/222237897-4561427d-b5de-4958-85d1-9f4be4ecec80.png)

i just see this rn, what it means
![image](https://user-images.githubusercontent.com/105376328/222240813-15e93e69-af83-4ed2-b282-1975c53dc4a5.png)

**Comment by Anmol-Baranwal** [comment, marked as answer]
GitHub pages don't take very long.

I recommend you to test it locally, read the guidelines [here](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll)
